courses.yml

```
@@ -504,3 +504,7 @@ lessons:
   url: https://github.com/mpavlase/naucse-python
   branch: compiled/feat/pyworking-gimp
   path: gimp
+2024/olomouc-pyladies-podzim:
```

```suggestion
2024/pyladies-olomouc-podzim:
```
Pro konzistenci s minulým během, plus už tak mám nachystané PR na web :laughing: